src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    private static final int MAX_ARRAY_SIZE = 10;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Random random = new Random();
+        int arraySize = random.nextInt(1, MAX_ARRAY_SIZE);
```

Comment:
Actually, can't get it why you decide to do this logic, we can just use: 
`Figure[] figures = new Figure[MAX_ARRAY_SIZE];`

You can write your thoughts in comment.